### ISSUE TITLE
app.HideHelp=true prints extraneous line at the usage string

### ISSUE THREAD

**Issue by ahmetalpbalkan** [codegangsta/cli#221]
### Before `app.HideHelp=true`:

```
NAME:
   [...]
GLOBAL OPTIONS:
   --help, -h		show help
   --version, -v	print the version

$
```

### After `app.HideHelp=true`:

```
NAME:
   [...]
GLOBAL OPTIONS:
   --version, -v	print the version


$
```

**Comment by jszwedko** [comment]
@ahmetalpbalkan I'm having trouble reproducing this behavior -- could you paste a full program?

**Comment by MartyMacGyver** [comment]
I'm seeing this as well... first, some sample code:

    package main
    
    import (
            "github.com/codegangsta/cli"
            "os"
    )
    
    func main() {
            app := cli.NewApp()
            app.Usage = "RabbitMQ stress tester"
            //app.HideVersion = true
            //app.HideHelp = true
            app.Action = func(c *cli.Context) {
                    //
            }
            app.Run(os.Args)
    }

With app.HideHelp = false, running this program (e.g., "foo -h") will show the expected help screen, followed by a blank line.

With app.HideHelp = true, running this program (e.g., "foo -h") actually throws an "Incorrect Usage." error, then prints the help with its blank line after, then prints yet another blank line.

**The extra blank lines being printed at line 96 and 104 in command.go, as well as line 208 of cli.go appear to be extraneous.**

**Comment by ahmetalpbalkan** [comment]
@martymacgyver nice find! Could you submit a PR?

**Comment by MartyMacGyver** [comment]
I've submitted pull request #249 to resolve this issue.

**Comment by jszwedko** [comment]
Closed via #249, thanks @MartyMacGyver !
